src/Downloader.php

```diff
@@ -128,21 +147,34 @@ protected function fetchCertificates(string $hostName): array
             'ssl' => $sslOptions,
         ]);
 
+        if ($this->usingIpAddress) {
+            $connectTo = $this->ipAddress;
+        } else {
```

Comment:
Let's avoid this `else` by using a ternary here.

Reply:
Thanks, done.